test/tests/functional/pbs_ralter.py

```diff
@@ -2842,3 +2842,42 @@ def test_restart_revert(self):
         self.server.expect(RESV, attrs, id=rid)
         wait = start - time.time()
         self.check_resv_running(rid, offset=wait)
+
+    def test_alter_degrade_reconfirm_standing(self):
+        """
+        Test that if a standing reservation is altered, degraded,
+        then reconfirmed, the reservation will use the original
+        select
+        """
+        duration = 60
+        offset = 60
+
+        self.server.manager(MGR_CMD_SET, SERVER, {'reserve_retry_time': 2})
```

Comment:
Use 5s.  2s means we'll try and reconfirm the resv every 2s.  We want to make sure there is enough time to have the cycle.  Also you don't want the reservation to be reconfirmed too quickly or your expects will fail.